pkgs/development/libraries/precice/default.nix

```diff
@@ -12,14 +12,21 @@ stdenv.mkDerivation rec {
   };
 
   preConfigure = ''
-    cmakeFlags="-DBUILD_SHARED_LIBS=ON -DPETSC=off"
+    cmakeFlags="-DBUILD_SHARED_LIBS=ON
```

Comment:
Could we convert this to `cmakeFlags` attribute passed to `mkDerivation`? See also #72074.